LumpNav/LumpNav.py

```diff
@@ -261,11 +267,16 @@ def setupScene(self): #applet specific
       self.needleModelToNeedleTip.SetName("NeedleModelToNeedleTip")
       m = vtk.vtkMatrix4x4()
       m.SetElement( 0, 0, 0 )
-      m.SetElement( 1, 1, 0 )

```

Comment:
There should be a settings selector when you start LumpNav (for example at the same place where we have the IP address selector now). We could use a simple string that would identify a child group in the ini file and settings would be read from there (e.g., LumpNav/Configurations/QueensHdhBreast, LumpNav/Configurations/MadridBreast) if a setting cannot be found in the specified child group then it would be read from the default group (LumpNav/Configurations/default). We can query the child groups in a group (http://doc.qt.io/qt-5/qsettings.html#childGroups) and populate a listbox with the available configurations.